orderer/common/multichannel/registrar.go

```diff
@@ -886,6 +908,11 @@ func (r *Registrar) joinSystemChannel(
 func (r *Registrar) RemoveChannel(channelID string) error {
 	r.lock.Lock()
 	defer r.lock.Unlock()
+
+	if removalStatus := r.pendingRemoval[channelID]; removalStatus {
+		return types.ErrChannelPendingRemoval
+	}
```

Comment:
please add to the unit tests of channelparticipation/restapi.go a test case in which the mock of the Registrar adds this error code and verify it is treated sensibly. e.g. restapi_test.go L:286, L:403, etc

Reply:
done. @tock-ibm do you agree on the statuses returned for removals and joins?